guides/Airline-Roadmap.md

```diff
@@ -0,0 +1,57 @@
+
+
+# The Airline Roadmap Structure
+
+
+
+  
+
+This section of the Cookbook defines our process for creating the roadmap for Airline.
+
+  
+
+## The Overall Goal of Airline
+
+  
+
+**Help our customers design, build and manage customer service automation at scale.**
+
+  
+
+This goal should be referred to throughout the roadmap process to make sure we prioritise and build the right feature.
+
+  
+
+## Airline Roadmap 2020
+
+  
+
+You can find the 2020 Airline Roadmap here.
+
+  
+
+[https://docs.google.com/spreadsheets/d/11aNJbQYyOEF2LWszwHLJzMIiCvry9gY46IfkrS67FKo/edit?usp=sharing](https://docs.google.com/spreadsheets/d/11aNJbQYyOEF2LWszwHLJzMIiCvry9gY46IfkrS67FKo/edit?usp=sharing)
+
+  
+
+All documentation related to Airline can be found in the [Airline Folder](https://drive.google.com/drive/folders/17qbwCq6uampCZkJUk8_Xw5Vvvv48iJIL?usp=sharing)
+
+  
+
+## How is the Airline Roadmap Defined
+
+  
+
+Goals are set on a quarterly basis by the Airline Product Manager based on the overall Airline Goal and discussion with the team.
+
+  
+
+Anyone in the company can suggest epics they think should be added to the airline roadmap on this [Trello board](https://trello.com/b/e1fAwJCw/airline-roadmap).
+
+  
+
+Once a month a group of people from across the business will review these epics and decide which should be added to the roadmap (and therefore the ‘In Planning’ list) and what their priority is. Based on this, in depth planning sessions will be organised for each new epic added to the ‘In Planning’ list. After each of these the roadmap will be updated by the Airline Product manager
```

Comment:
'In Planning' -> `In Planning`. also missing a full-stop at the end of the sentence.